### DISCUSSION TITLE
Random classes just don't work, is that only me? (RN)

### DISCUSSION THREAD

**Discussion by marcelino-borges** [nativewind/nativewind#1116] [General]
Hey guys!

I'm just wondering if I'm doing anything the wrong way, but I often find random classes in random components in React Native not working... Right now I'm using Native Wind in a brand new React Native Expo project:

```
"expo": "~51.0.28",
"nativewind": "4.0.36",
"react": "18.2.0",
"react-native": "0.74.5",
"tailwindcss": "^3.4.13",
"typescript": "~5.3.3"
```

So far it looks literally random, I haven't noticed any nexus to anything specific. BTW right now I'm really considering to stop using native wind and go back to the old fashion RN stylesheets =(

I'm pretty used to have tailwindcss in NextJS projects and it all works like a charm, but using NativeWind in React Native is tough...
Not rarely I need to use something into `style={}`, because a NW class doesn't take effect. 

Sometimes restarting the app fixes the problem, sometimes it doesn't...

It's really slowing development down.

Is that only here?
Is there anything I should know in order to avoid these issues?

I didn't event consider to open an issue, because I'm not sure if I could reproduce consistently what I'm experiencing

**Comment by elalami-m** [comment]
I'm facing the same problems in nativewind v4

**Comment by marcelino-borges** [comment]
Sadly no one from NW team to help or comment

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
